src/extra/nnue_data_binpack_format.h

```diff
@@ -6835,6 +6845,17 @@ namespace binpack
         {
             return pos.isMoveLegal(move);
         }
+
+        [[nodiscard]] bool isCapturingMove() const
+        {
+            return pos.pieceAt(move.to) != chess::Piece::none() &&
```

Comment:
do you want to deal with en passant with this ?

Reply:
Hmm, indeed we don't consider en passant a capture, during training too. If anything, should be investigated separately and applied in both places.